src/pages/courses/[vuid].tsx

```diff
@@ -83,16 +83,24 @@ export async function getStaticPaths() {
     `${process.env.STRAPI_API_URL}/courses`
   )
 
-  const paths = courses.data.data.map((course) => {
-    return {
-      params: { id: `${course.id}` },
-    }
-  })
+  console.log(courses.data.data)
```

Comment:
Remove console.log?

Reply:
Nice catch. I removed it in a new commit, where I also changed the getStaticPaths fallback to 'blocking' instead of false, so we can get newly published entries to get rendered as well.
false simply declines all paths that were not present in the initial build.